packages/block-library/src/rss/block.json

```diff
@@ -39,6 +39,18 @@
 		"excerptLength": {
 			"type": "number",
 			"default": 55
+		},
+		"openInNewTab": {
+			"type": "boolean",
+			"default": false
+		},
+		"addNofollow": {
+			"type": "boolean",
+			"default": false
+		},
+		"additionalRelAttributes": {
+			"type": "string",
+			"default": ""
```

Comment:
I think there's no need to have two separate attributes for `rel`. Maybe we should just display single text control, using a single attribute (`linkRel`) and let people input `nofollow` or any additional field as needed.